terraform/modules/common/ami/main.tf

```diff
@@ -0,0 +1,33 @@
+## Variables
+
+variable "amazon_release" {
+  description = "The amazon release used to generate the encrypted ami"
+}
+
+variable "stack_name" {
```

Comment:
Do we need the variable in this file given it's not used directly in it?

Reply:
Can probably take out